server/scores.js

```diff
@@ -4,27 +4,80 @@ const express = require('express')
 const Promise = require('bluebird')
 const { MongoClient, ObjectID } = require('mongodb')
 const { authroizationMiddlware } = require('@first-lego-league/ms-auth')
+const Configuration = require('@first-lego-league/ms-configuration')
 
 const DEFAULTS = require('./defaults')
 
 const mongoUrl = process.env.MONGO_URI || DEFAULTS.MONGO
 
 const router = express.Router()
 
+const STATUS = {

```

Comment:
I like the constants, but why aren't you using exceptions like I asked for before? It's ok if you have a reason, but please respond to the comment in the conversation area. Tenet, just because something is marked "outdated"it didn't mean it is